_amenities/pools/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Klahanie Pools
+date: 2019-05-28 05:11
+author: Klahanie
+comments: true
+categories: [pool]
+show-title: true
+---
+<div class="alert alert-warning clearfix">Note: Klahanie Pools are closed due to the COVID-19 pandemic until further notice.
+</div>
+
+<img src="{{site.url}}images/swimming1.jpg" class="float-right col-sm-4 img-thumbnail">
+
+* [Lakeside Pool Schedule]({{site.url}}amenities/pools/lakeside-pool.html)
+* [Mountainview Pool Schedule]({{site.url}}amenities/pools/mountainview-pool.html)
+* [Swim Lessons]({{site.url}}amenities/pools/swim-lessons.html)
+* [Pool rules]({{site.url}}amenities/pools/pool-rules.html)
+
+
+The Klahanie Association operates two 25 meter Pools with 4 lanes. Lakeside Pool is open all year round. Mountainview Pool operates from Memorial Weekend until  Labor day. Whether you are looking for pool schedule, swim lessons, pool employment, pool rules and more, you'll find detailed information below.

```

Comment:
"two 25-meter pools, each with four lanes. Lakeside Pool is open all year."
"until Labor Day." (eliminate extra space and add a capital)